docs/design/storagenode-graceful-exit-ui.md

```diff
@@ -0,0 +1,53 @@
+# Storage Node Graceful Exit - User Interface
+
+## Abstract
+
+A storage node operator needs the ability to request a Graceful Exit on a per satellite basis.
+This document describes how the storage node operator manages the Graceful Exit process.
+
+## Background
+
+The storage node operator needs:
+- a way to initiate graceful exit, 
+- a way to monitor graceful exit progress, and
+- terminate graceful exit without escrow.
```

Comment:
Is this an all-or-nothing deal? Do we have plans and/or can we pay nodes back the difference between the amount in escrow and the cost of repairing an ungraceful exit?

Reply:
:+1: